shinysdr/telemetry.py

```diff
@@ -109,21 +109,26 @@ def __new__(cls, *args, **kwargs):
 class ITelemetryObject(Interface):
     """
     An object that can be in an TelemetryStore.
+
+    Examples: an APRS station, aircraft, etc. The state of an ITelemetryObject
+    is updated through receiving ITelemetryMessage.
     """
-    
+
     def receive(message):
         """
-        TODO document
+        Update state according to the received ITelemetryMessage.
         """
-    
+
     def is_interesting():
         """
-        Return whether this object should be shown to the client. The value should change only when a message is receive()d.
+        Return whether this object should be shown to the client. The value
+        should change only when a message is receive()d.
```

Comment:
Your own written style guidelines say to follow PEP8:

> Limit all lines to a maximum of 79 characters.
>
>For flowing long blocks of text with fewer structural restrictions (docstrings or comments), the line length should be limited to 72 characters.

And as an aside, it's no fun as a contributor to work on a patch and then get a bunch of style comments which require a bunch of tedious editing. I'm not one at all to argue style doesn't matter, but that's why there automated tools (like pep8/flake8) to address exactly this problem. Set up Travis or similar to run it automatically for any PR in case the author neglects to do so. If you have style requirements beyond what pep8 provides, then find or create a program to enforce those.